xsd/siri_model/siri_facility-v2.0.xsd

```diff
@@ -721,4 +721,37 @@ Rail transport, Roads and road transport
    </xsd:sequence>
   </xsd:sequence>
  </xsd:group>
+<!-- ===Vehicle FACILITY====================================== -->
+<xsd:complexType name="TypeOfVehicleFuelStructure">
+	<xsd:annotation>
+		<xsd:documentation>Classification of Vehicle Energy type. See Transmodel TypeOfFuel</xsd:documentation>
+	</xsd:annotation>
+	<xsd:sequence>
+		<xsd:element name="FuelType" type="TypeOfVehicleFuelEnumeration">
```

Comment:
Please change type to the existing "TypeOfFuelEnumeration" of the siri_journey_support schema